traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -207,6 +207,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodPost, `users/?(\.json)?$`, api.CreateHandler(&user.TOUser{}), auth.PrivLevelOperations, Authenticated, nil},
 
 		{1.1, http.MethodGet, `user/current/?(\.json)?$`, user.Current, auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.1, http.MethodPut, `user/current(/|\.json)?$`, user.ReplaceCurrent, auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
is readonly the right priv

Reply:
Yeah, Perl didn't check any permissions role for this endpoint. I actually think that's why it exists, it allows read-only users to edit their own information without needing to add a check for that in the `PUT /users` handler.